staging/src/k8s.io/api/core/v1/annotation_key_constants.go

```diff
@@ -156,4 +156,20 @@ const (
 	// heuristics will often populate topology hints on EndpointSlices, but that
 	// is not a requirement.
 	AnnotationTopologyMode = "service.kubernetes.io/topology-mode"
+
+	// MountOptionAnnotation can be used for a persisent volume claim and
+	// a persisent volume to specify the name of the storage class. This annotation has
+	// been deprecated in favor of the `storageClassName` field in the PVC and PV spec.
```

Comment:
incorrect comment here?

Reply:
Fixed: renamed to BetaStorageClassAnnotation